app/controllers/access_requests_controller.rb

```diff
@@ -7,10 +7,11 @@ def approve
     id = params[:id]
 
     begin
-      AccessRequest.find(id).approve!
+      access_request = AccessRequestAPI.new(id: id)
+      access_request.approve(id: id)
       flash[:notice] = "Successfully approved request"
       redirect_to action: 'inform_publisher', id: id
-    rescue ManageCoursesAPI::AccessRequestInternalFailure => e
+    rescue StandardError => e
```

Comment:
I think this will prevent the error from propagating up to Sentry, can we add a `Raven.capture(e)` here?

Reply:
done, thanks for the snippet :+1: